src/common/pubsub.ts

```diff
@@ -246,3 +267,28 @@ export const notifySourceFeedRemoved = async (
     feed,
     sourceId,
   });
+
+export const notifySubmissionChanged = async (
+  log: EventLogger,
+  submission: ChangeObject<Submission>,
+): Promise<void> => {
+  if (submission.status === SubmissionStatus.Rejected) {
+    publishEvent(log, communityLinkRejectedTopic, submission);
+  }
```

Comment:
This should not include any business logic. The business logic should be in the calling function